vispy/scene/widgets/widget.py

```diff
@@ -224,21 +244,21 @@ def add_grid(self, *args, **kwargs):
         """
         Create a new Grid and add it as a child widget.
 
-        All arguments are given to add_widget().
+        All arguments are given to Grid().
```

Comment:
I think that was because `Grid.add_widget()` does accept arguments (row, col, etc.)